cpg_workflows/stages/gatk_sv/gatk_sv_multisample.py

```diff
@@ -0,0 +1,1328 @@
+"""
```

Comment:
this is the result of smashing together multisample_1 (619 lines) and multisample_2 (911 lines) into a continuous workflow. Looks heinous, isn't really